src/Funds/Funds.js

```diff
@@ -21,6 +22,10 @@ const Funds = () => {
         path={`${FUNDS_ROUTE}/edit/:id`}
         component={EditFund}
       />
+      <Route
+        path={`${FUNDS_ROUTE}/view/:id/budget/:budgetId/transactions`}
```

Comment:
what was the idea behind of putting this route here?

Reply:
to keep funds routes (under `/finance/funds`) in one place

Reply:
but it's the same url as in https://github.com/folio-org/ui-finance/blob/master/src/components/Budget/Budget.js#L31 , no?

Reply:
`finance/budget/:id/transactions` vs `finance/funds/view/:id/budget/:budgetId/transactions`

Reply:
it's hard to understand at first glance. Maybe it makes sense to move transaction routes under `Transactions` and include them into `Main`

Reply:
Moved `Transactions` to `Main`